README.md

```diff
@@ -31,6 +31,11 @@ Documentation for the API was created in OpenAPI format and is hosted [here](htt
 
 - add better error handling when dealing with response data (display error message when something will fail)
 - add caching to avoid server querying with every location change (if location was visited then keep it in memory and use it again)
+- add instruction how to run project locally with Docker
+
+##### Front-End Todo's
+
+- All class components might be refactored to functional components
```

Comment:
Can we move this under Todo's section and mark it with `[React]` label like this:

`- [React] all class components might be refactored to functional components`

I already have `React` things to improve there, label is missing (will fix that) 😄